From 42da4086b987fbb35562e93e534e57ad3f81f855 Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Wed, 17 Mar 2021 16:55:28 +0900
Subject: [PATCH] cifsd: fix WARNING: document isn't included in any toctree
Git-commit: 42da4086b987fbb35562e93e534e57ad3f81f855
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Stephen reported a warning message from cifsd.rst file.

 Documentation/filesystems/cifs/cifsd.rst: WARNING: document isn't
 included in any toctree

Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 Documentation/filesystems/index.rst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/filesystems/index.rst b/Documentation/filesystems/index.rst
index d4853cb919d2..ac9396f2bb8a 100644
--- a/Documentation/filesystems/index.rst
+++ b/Documentation/filesystems/index.rst
@@ -72,6 +72,7 @@ Documentation for filesystem implementations.
    befs
    bfs
    btrfs
+   cifs/cifsd
    cifs/cifsroot
    ceph
    coda
-- 
2.33.0

